configuration/containers.config

```diff
@@ -14,6 +14,21 @@ process {
   $BuildVCFIndex.container            = "${params.repository}/igvtools:${params.tag}"
   $ConcatVCF.container                = "${params.repository}/sarek:${params.tag}"
   $CreateRecalibrationTable.container = "${params.repository}/gatk:${params.tag}"
+  $GetVersionAll.container            = "${params.repository}/qctools:${params.tag}"
```

Comment:
It may be possible to tidy up the configs here quite a bit with the new nextflow concept of [`label`](https://www.nextflow.io/docs/latest/process.html#process-label) (in new versions of NF only)